example-react-app/src/app/AppMenu.tsx

```diff
@@ -16,205 +16,205 @@ export const AppMenu = () => {
         screenId={"TestBlankComponent"}
         icon={<BarsOutlined />}
         caption={"menu.TestBlankComponent"}
-        key={"1214a4b6-7bf9-4f9b-ae4d-0593b09562a7"}
+        key={"63b4027b-99c1-4136-8367-01372551aac9"}
```

Comment:
Let's do not include this file in the changeset if there are no meaningful changes